app/controllers/api/v1/users_controller.rb

```diff
@@ -23,7 +23,8 @@ def create
 
         if user.save
           session[:user_id] = user.id
-          render_json status: :created
+          token = user.generate_activation_token!
+          render_json data: { token: }, status: :created # TODO: enable activation email sending.
```

Comment:
Why are we returning the token from the user create?

Reply:
When we're supposed to send an email with the token I'm sending the token in response.
Shouldn't we send activation token on create?

Reply:
Until we setup the email how could we test the reset password and account activation if we're not returning the token?
The only time GL will have the token is on generating or receiving it.
Do you like if we log it to the console instead?

Reply:
Yup, I'm aware of it that's why I left the inline todo comments `# TODO: enable activation email sending.`